environment.yml

```diff
@@ -0,0 +1,17 @@
+name: mhkit-docs
+channels:
+  - conda-forge
+  - defaults
+dependencies:
+  - python=3.10
+  - sphinx=8.1.3
+  - pip
+  - pandoc
+  - ipython
+  - pip:
+      - sphinxcontrib-bibtex==2.6.3
+      - sphinxcontrib-matlabdomain==0.22.1
+      - sphinx-rtd-theme==3.0.2
+      - nbsphinx==0.9.5
+      - sphinxcontrib-googleanalytics==0.4
+      - -e ./MHKiT-Python
```

Comment:
Cool, I didn't realize one could add an editable local install to an environment file

Reply:
There's currently a `docs/environment.yml` file. Remove one or the other